app/Policies/UserPolicy.php

```diff
@@ -45,15 +45,15 @@ class UserPolicy
     public function viewAny(Authenticatable $authenticatable): Response
     {
         return $authenticatable->canOrElse(
-            abilities: 'user.view-any',
+            abilities: ['user.view-any', 'license.create'],
             denyResponse: 'You do not have permission to view users.'
         );
     }
 
     public function view(Authenticatable $authenticatable, User $model): Response
     {
         return $authenticatable->canOrElse(
-            abilities: ['user.*.view', "user.{$model->id}.view"],
+            abilities: ['user.*.view', "user.{$model->id}.view", 'license.create'],
```

Comment:
No need to do this here. Just having this license permission on the field is fine.